.github/workflows/ci.yml

```diff
@@ -965,3 +965,59 @@ jobs:
           inputs: '{"version":"canary","environment":"staging","version_extra":"${{ needs.job_setup.outputs.branch_name }}"}'
           wait-for-completion-timeout: 25m
           wait-for-completion-interval: 30s
+
+  publish_admin_x_activitypub:
+    needs: [
+      job_setup,
+      job_lint,
+      job_unit-tests
+    ]
+    name: Publish @tryghost/admin-x-activitypub
+    runs-on: ubuntu-latest
+    if: always() && needs.job_setup.result == 'success' && needs.job_lint.result == 'success' && needs.job_unit-tests.result == 'success'
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '18.12.1'
+
+      - name: Restore caches
+        uses: actions/cache@v4
+        with:
+          path: |
+            ${{ github.workspace }}/apps/admin-x-activitypub/node_modules
+          key: admin-x-activitypub-${{ github.sha }}
+          restore-keys: |
+            admin-x-activitypub-
+
+      - name: Install dependencies
+        run: yarn install --prefer-offline --frozen-lockfile
+
+      - name: Build the package
+        run: yarn run nx build @tryghost/admin-x-activitypub
+
+      - name: Check if version changed
+        id: version_check
+        working-directory: apps/admin-x-activitypub
+        run: |
+          CURRENT_VERSION=$(cat package.json | jq -r .version)
+          PUBLISHED_VERSION=$(npm show @tryghost/admin-x-activitypub version || echo "0.0.0")
+          echo "Current version: $CURRENT_VERSION"
+          echo "Published version: $PUBLISHED_VERSION"
+          if [ "$CURRENT_VERSION" = "$PUBLISHED_VERSION" ]; then
+            echo "Version is unchanged."
+            echo "version_changed=false" >> $GITHUB_ENV
+          else
+            echo "Version has changed."
+            echo "version_changed=true" >> $GITHUB_ENV
+          fi
+
+      - name: Publish to npm
+        if: env.version_changed == 'true'
+        working-directory: apps/admin-x-activitypub
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+        run: npm publish --dry-run --access public
```

Comment:
One thing we might want to do after this is to auto-purge the jsDelivr cache

Reply:
Yeah, I was thinking this too